eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/grpc/consumer/EventMeshGrpcConsumer.java

```diff
@@ -130,7 +144,7 @@ public void subscribe(List<SubscriptionItem> subscriptionItems) {
 
     private void addSubscription(List<SubscriptionItem> subscriptionItems, String url) {
         for (SubscriptionItem item : subscriptionItems) {
-            subscriptionMap.put(item.getTopic(), new SubscriptionInfo(item, url));
+            subscriptionMap.putIfAbsent(item.getTopic(), new SubscriptionInfo(item, url));
```

Comment:
Shall we update the subscription info if the topic exists?

Reply:
ok,I fix it. @horoc

Reply:
> ok,I fix it. @horoc

@jonyangx  seems it modified to ```putIfAbsent```  here again in the later commit  ``` 5d7e502 :
fix license check error```